source/faq/faq.rst

```diff
@@ -325,8 +309,10 @@ device `3bb9`:
       }
     ]
 
-There are more operators that could be used to filter entries.
-Check `History API <https://dojot.github.io/history/apiary_latest.html>`_
+Check a tutorial using history in :ref:`Checking historical data`.
```

Comment:
Suggestion:
For more details on data retrieval from the history, check the <link> tutorial.